### ISSUE TITLE
Thumbnails in Browse view only display text "Thumbnail"

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#662]
In the masters theses collection, there are some items, e.g., http://hdl.handle.net/10919/81752, with a good jpg thumbnail that appears in Simple Item View. But 'Thumbnail' text appears in Browse. I recreated these thumbnails but the issue remains.

**Comment by alawvt** [comment]
Theses with this problem:

- [x] http://hdl.handle.net/10919/78197
- [x] http://hdl.handle.net/10919/78889
- [x] http://hdl.handle.net/10919/81752
- [x] http://hdl.handle.net/10919/83450
- [x] http://hdl.handle.net/10919/83597
- [x] http://hdl.handle.net/10919/83759
- [x] http://hdl.handle.net/10919/83922
- [x] http://hdl.handle.net/10919/87059

**Comment by alawvt** [comment]
https://groups.google.com/forum/#!searchin/dspace-tech/thumbnails$20do$20not$20show$20up$20in$20browse%7Csort:date/dspace-tech/4K-MUXVGeGA/Oku02BAdAQAJ

George,

Thank you for the clue about the primary bitstream. As reported in https://github.com/VTUL/vtechworks/issues/662, we have some items that have only the text "Thumbnail" in Browse, rather than the thumbnail, itself. However the thumbnail does appear correctly in Simple Item View for these items. I tried changing these [ORIGINAL] files to 'primary' and that made the thumbnail appear correctly in Browse. I then changed them back to 'not primary' and the thumbnail still appeared correctly in Browse. We have thousands of items that are primary and not primary and their thumbnails appear correctly in Browse. Why these got this way, I don't understand but I do appreciate your help toward a fix.
